### PR TITLE
Reveal openshift_master_manage_htpasswd variable in upgrade preparations

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1666644

The goal of this PR is to address the issue described in the bug linked above and described succinctly in this[0] solution. I'd appreciate suggestions on better ways to surface this information. I included it under 3. rather than 2. because it affects upgrading z-stream releases as well.

Thank you.

[0] https://access.redhat.com/solutions/3811271

### CODE REVIEW

**Comment by jboxman** [comment]
@openshift/team-documentation PTAL, thanks.

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -247,6 +248,10 @@ you used in previous versions.
 You must not remove the `openshift_kubelet_name_override` parameter from your
 inventory file after you upgrade.
 ====
+.. If you manage *_/etc/origin/master/htpasswd_* manually, set
```
Comment (kalexand-rh) [inline upgrading/automated_upgrades.adoc]: I'd say:

If you manually manage the cluster's  \*_/etc/origin/master/htpasswd\_* file, add
\`openshift_master_manage_htpasswd=false\` to your inventory file to prevent the
upgrade process from overwriting the \*_htpasswd\_* file.

**Review by kalexand-rh — COMMENTED** [review]
I'd reword the change, but it LGTM

File: upgrading/automated_upgrades.adoc (hunk 1 above)
Comment (jboxman) [reply upgrading/automated_upgrades.adoc]: I like that, thanks!

**Comment by jboxman** [comment]
@openshift/team-documentation, this PR is approved in the BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1666644#c8

If that is sufficient, I can merge this. Thanks.

**Comment by kalexand-rh** [comment]
@jboxman, yup! After you make updates from peer review and QE review, you're good to merge and cherry-pick.

**Comment by jboxman** [comment]
Verified by QE[0].

[0] https://bugzilla.redhat.com/show_bug.cgi?id=1666644#c10
